specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -0,0 +1,4472 @@
+          "type": "string",
+          "description": "A string containing further details about the install result"
+        }
+      },
+      "required": [
+        "resultCode",
+        "extendedResultCode"
+      ],
+      "description": "The step result under an update."
+    },
+    "InstallResult": {
+      "type": "object",
+      "properties": {
+        "resultCode": {
+          "type": "integer",
+          "description": "Install result code."
+        },
+        "extendedResultCode": {
+          "type": "integer",
+          "description": "Install extended result code"
+        },
+        "resultDetails": {
+          "type": "string",
+          "description": "A string containing further details about the install result"
+        },
+        "stepResults": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StepResult",
+            "description": "Step results"
+          },
+          "description": "Array of step results"
+        }
+      },
+      "required": [
+        "resultCode",
+        "extendedResultCode"
+      ],
+      "description": "The install result of an update and any step results under it."
+    },
+    "DeviceUpdateAgentId": {
+      "type": "object",
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "Device Id"
+        },
+        "moduleId": {
+          "type": "string",
+          "description": "Module Id"
+        }
+      },
+      "required": [
+        "deviceId"
+      ],
+      "description": "Device Update agent id"
+    },
+    "LogCollection": {
+      "type": "object",
+      "properties": {
+        "operationId": {
+          "type": "string",
+          "description": "The log collection id.",
+          "x-ms-client-name": "logCollectionId"
+        },
+        "deviceList": {
+          "type": "array",
+          "description": "Array of Device Update agent ids",
+          "items": {
+            "$ref": "#/definitions/DeviceUpdateAgentId"
+          }
+        },
+        "description": {
+          "type": "string",
+          "maxLength": 512,
+          "description": "Description of the diagnostics operation."
+        },
+        "createdDateTime": {
+          "type": "string",
+          "description": "The timestamp when the operation was created.",
+          "readOnly": true
+        },
+        "lastActionDateTime": {
+          "type": "string",
+          "description": "A timestamp for when the current state was entered.",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/OperationStatusWithoutUndefinedOption",
+          "description": "Operation status.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "deviceList"
+      ],
+      "description": "Diagnostics request body"
+    },
+    "LogCollectionOperationDetailedStatus": {
+      "type": "object",
+      "properties": {
+        "operationId": {
```

Comment:
Here too I think this property should be named "logCollectionId".

Reply:
See: https://github.com/Azure/azure-rest-api-specs/pull/20023#discussion_r939428810